addons/project/models/project.py

```diff
@@ -773,7 +773,8 @@ def action_view_tasks(self):
         context = ast.literal_eval(context)
         context.update({
             'create': self.active,
-            'active_test': self.active
+            'active_test': self.active,
+            'allow_milestones': self.allow_milestones,
```

Comment:
If the action was just a xml action we cannot do that.

Can you remove this commit we will not do that because the generic behavior is we can change the visibility of the filter if the global settings is enabled or not?

Adding some additional context just to change the visible of a filter is not really a good idea.

Thanks for your understanding. 🙂

Reply:
> If the action was just a xml action we cannot do that.
> 
> Can you remove this commit we will not do that because the generic behavior is we can change the visibility of the filter if the global settings is enabled or not?
> 
> Adding some additional context just to change the visible of a filter is not really a good idea.
> 
> Thanks for your understanding. 🙂

Ok but already based on the context so I do. [here](https://github.com/odoo/odoo/blob/16.0/addons/sale/report/sale_report_views.xml#L80)